### DISCUSSION TITLE
Can I install kong KIC with kong manger?

### DISCUSSION THREAD

**Discussion by Imccccc** [Kong/kubernetes-ingress-controller#4416] [Q&A]
I installed kong KIC in my k8s cluster, but monitoring and global rule view are necessary. Could I deploy a Kong manager for KIC?

**Comment by randmonkey** [comment]
Yes, you can install both, but you cannot configure Kong by Kong manager. The Kong configuration would be totally controlled by KIC. Because the reconcile loop of KIC will overwrite Kong configuration in a certain sync interval.
